bin/auto-cpufreq

```diff
@@ -86,7 +87,9 @@ def main(config, daemon, debug, install, remove, live, log, monitor, stats, vers
                     set_autofreq()
                     countdown(2)
             else:
-                daemon_not_found()
+                pass
+            #"daemon_not_found" is not defined
```

Comment:
Let's remove this comment.

Reply:
Another good catch, not good to have calls to functions which do not exist.

Reply:
thanks, leaving it commented and resolving the conversation